website/community/index.mdx

```diff
@@ -72,6 +64,29 @@ Enhancements or new feature proposals are also welcome. The more concrete and ra
 
 * OpenDAL core repository: https://github.com/apache/incubator-opendal
 
+## Upcoming Community Events
+
+For now, we have a single community event known as the Community Sync Meeting.
+This meeting is open to everyone.
+
+During the meeting, we will cover various topics including project status,
+roadmap updates, and other relevant discussions.
+We will also have a Q&A session at the end of the meeting.
+
+Attending the meeting presents a valuable opportunity
+for you to meet fellow community members and gain deeper insights into the project.
+Moreover, it provides an excellent chance for active involvement in the project's activities.
+
+We currently organize the community sync meeting every three weeks.
+The meeting takes place at 14:00 UTC+8 on the Friday of the final week in each three-week cycle.
```

Comment:
The time has been changed. Would you like to change it in this PR?

Reply:
Right